habitat/config/default.py

```diff
@@ -216,7 +516,14 @@ def __init__(self, *args, **kwargs):
 _C.SIMULATOR = CN()
 _C.SIMULATOR.TYPE = "Sim-v0"
 _C.SIMULATOR.ACTION_SPACE_CONFIG = "v0"
+_C.SIMULATOR.CREATE_RENDERER = True
```

Comment:
@aclegg3, @ASzot is that right default value for Habitat Lab? It looks relevant only for Habitat 2.0 tasks and possibly need to be disabled by default. As well as other values like step physics, update robot, concur render?

Reply:
For some reason we get error for example_test on CI:
```
[17:26:07:066416]:[Physics] PhysicsManager.cpp(244)::addObject : Simplified template handle : 003_cracker_box  | newObjectHandle : 003_cracker_box_:0000
GL::Context: unsupported version string: 3.0.0 NVIDIA 418.87.00
BackgroundRenderer: Failed to create OpenGL context
```

Reply:
The culprit was CONCUR_RENDER that I set to True instead of False. 
I changed it and the test is now passing. 
I also changed AUTO_SLEEP to False by default since the code seems to be [using False by default](https://github.com/facebookresearch/habitat-lab/blob/main/habitat/tasks/rearrange/rearrange_sim.py#L50).